tools/cdb2sql/cdb2sql.cpp

```diff
@@ -175,8 +176,31 @@ const char *words[] = {
   "SELECT", "SELECTV", "SET",
   "TRUNCATE",
   "UPDATE",
-  "WITH", NULL, }; // must be terminated by NULL
+  "WITH", NULL,  // must be terminated by NULL
+};
+
+
+const char *char_alpha_words[] = {
```

Comment:
Could you explain this array? The name's not very clear.

Reply:
alpha==@ in my mind. did not want to call it char_at_words[]. I can rename it to char_atglyph_words[] maybe.